flake8_implicit_str_concat.py

```diff
@@ -1,10 +1,12 @@
 """
 Forbid implicitly concatenated string literals on one line such as those
-introduced by Black
+introduced by Black.
+Forbid all explicitly concatenated strings.
 """
```

Comment:
At https://pypi.org/project/flake8_implicit_str_concat/ the one-line summary is cut short:

![image](https://user-images.githubusercontent.com/1324225/69804100-7ddce400-11e6-11ea-9c7f-e8913bf0a40c.png)

https://github.com/takluyver/flit#usage says

> Make sure that foobar's docstring starts with a one-line summary of what the module is

So perhaps add a short one-liner on the first line?

Reply:
@hugovk https://github.com/keisheiled/flake8-implicit-str-concat/issues/11